content/strategy-goals/strategy/code-graph/search/core.md

```diff
@@ -12,22 +12,27 @@ Quicklinks:
 
 ## Where we are now
 
-In FQ3, we grew the Sourcegraph Cloud global index to 2.1M repositories, including all repositories with 6 stars or more. Importantly, the changes we’ve made to reach this state have never been cloud-specific, and yielded trickle-down benefits to all Sourcegraph deployments (for instance, [significant reductions](https://about.sourcegraph.com/blog/zoekt-memory-optimizations-for-sourcegraph-cloud/) in the memory usagee of Zoekt, our trigram-based indexed search backend).
+In **FY22Q3**, we grew the Sourcegraph Cloud global index to 2.1M repositories, including all repositories with 6 stars or more. Importantly, the changes we’ve made to reach this state have never been cloud-specific, and yielded trickle-down benefits to all Sourcegraph deployments (for instance, [significant reductions](https://about.sourcegraph.com/blog/zoekt-memory-optimizations-for-sourcegraph-cloud/) in the memory usagee of Zoekt, our trigram-based indexed search backend).
+
+In **FY22Q4**, we conducted discovery work to better understand the bottlenecks of our search infrastructure on large monorepos (>6GB working directory). We also started growing our search index to include repositories from more non-GitHub.com and GitLab.com code hosts: for instance, you can now search [34k repositories from src.fedoraproject.org on sourcegraph.com](https://sourcegraph.com/search?q=context:global+r:%5Esrc%5C.fedoraproject%5C.org/+type:repo+count:all&patternType=literal).
 
 ## What's next and why
 
-### FY22Q4
+### FY23Q1
 
 #### Goals
 
-- Make Sourcegraph Search work well with large-scale monorepos (time to first result for unindexed search on monorepos up to 60GB is < 10s and <2s for indexed, and time to indexing is always <30 minutes).
-- Sourcegraph Cloud indexes public repositories globally from 4 non-GitHub code hosts.
+- **Monorepo performance**: At a P75 level, a synthetic repo of 15GB will index in less than 30 minutes, indexed searches complete in < 2s and unindexed searches complete in < 10s
+- **Ranking**: We start tracking ranking quality, using selected search results as a proxy.
+- **Code host coverage**: Sourcegraph Cloud indexes public repositories globally from the most popular package hosts.
 
 #### Details
```

Comment:
This seems to be more of an explanation of why rather than more specific details, maybe list the reason why along with the goals in the above section? Or rename this to something like "Reasoning"